File: src/Systems/Fortran/patterns/client_server.f90
module client_server_example
  implicit none
contains
  logical function run()
    character(len=32) :: response
    response = server('sku-1')
    run = trim(response) == '200:stock=7'
  contains
    function server(key) result(text)
      character(len=*), intent(in) :: key
      character(len=32) :: text
      if (key == 'sku-1') then; text = '200:stock=7'; else; text = '404:missing'; end if
    end function
  end function
end module

File: src/Systems/Swift/patterns/ClientServer.swift
enum ClientServerExample {
    static func run() -> Bool {
        let server: (String) -> (Int, String) = {
            $0 == "sku-1" ? (200, "stock=7") : (404, "missing")
        }
        let response = server("sku-1")
        return response.0 == 200 && response.1 == "stock=7"
    }
}
